scripts/tmuxinator.sh

```diff
@@ -12,6 +12,9 @@ if [[ -z "${IN_NIX_SHELL:-}" ]]; then
   sleep 3
 fi
 
+# Flag to enable verbose build output from depndent processes (disabled by default)
+export FM_VERBOSE_OUTPUT=0
```

Comment:
Why set it by default here? Doesn't this just make it harder to activate verbose output by requiring a file edit? OTOH it might make the flag more discoverable?

Reply:
Yeah. Command line arguments would have been better. But it seemed more involved as they clashed with the FM_FED_SIZE variable further down.